metron-platform/metron-elasticsearch/pom.xml

```diff
@@ -206,6 +220,16 @@
                             <goal>shade</goal>
                         </goals>
                         <configuration>
+                            <relocations>
+                                <relocation>
+                                    <pattern>com.google.common</pattern>
+                                    <shadedPattern>org.apache.metron.guava.metron-elasticsearch</shadedPattern>
+                                </relocation>
```

Comment:
Are we shading the wrong jar here?  Shouldn't this be `<pattern>com.google.guava</pattern>`?